includes/class-coblocks-form.php

```diff
@@ -680,6 +698,18 @@ public function process_form_submission( $atts ) {
 			$post_id
 		);
 
+		$short_code_regex = '/\[(.*?)\]/i';
+		if ( preg_match_all( $short_code_regex, $subject, $matches_out ) ) {
+			foreach ( $matches_out[0] as &$value ) {
```

Comment:
@paranoia1906 are we sure `$matches_out[0]` could never throw an undefined index notice?

Reply:
I was not able to get an undefined index here. Since the `$matches_out[0]` is conditional to a 'truthy' `preg_match_all` execution. This returns null without a match.

Reply:
I think this covers all options a user may try. Let me know if I am off track.